packages/python/src/version.ts

```diff
@@ -90,3 +116,10 @@ function isDiscontinued({ discontinueDate }: PythonVersion): boolean {
   const today = Date.now();
   return discontinueDate !== undefined && discontinueDate.getTime() <= today;
 }
+
+function isInstalled({ pipPath, pythonPath }: PythonVersion): boolean {
+  return (
+    !!which.sync(pipPath, { nothrow: true }) &&
+    !!which.sync(pythonPath, { nothrow: true })
```

Comment:
```suggestion
    Boolean(which.sync(pipPath, { nothrow: true })) &&
    Boolean(which.sync(pythonPath, { nothrow: true }))
```

Nit pick: this will avoid the double negative. I'm also thinking this should be async parallel instead with `await Promise.all()`

Reply:
I've replaced the `!!` with a `Boolean()`, but I don't really agree. Calling the Boolean constructor costs more than a simple double negation.

I don't think it's not worth parallelizing these two stats. Using `Promise.all()` would require `isInstalled()` to return the promise and since `isInstalled()` is being called from a `Array.prototype.find()`. I'd have to replace the `find()` with more code and a for loop. With all that overhead and synchronization, two stat calls doesn't seem half bad.